doc/modules/ROOT/pages/ops.adoc

```diff
@@ -59,7 +59,7 @@ Returns::
 
 === `:eval`
 
-Evaluates code.
+Evaluates code. Note that unlike regular stream-based Clojure REPLs, nREPL's `:eval` short-circuits on first read error and will not try to read and execute the remaining code in the message.
```

Comment:
Btw, that’s an auto-generated page, so you can’t edit it directly. You have to edit the op’s description and regen the page from it.

Reply:
How to regen?